homeassistant/components/group/light.py

```diff
@@ -324,6 +379,21 @@ async def async_update(self):
             effects_count = Counter(itertools.chain(all_effects))
             self._effect = effects_count.most_common(1)[0][0]
 
+        self._color_mode = None
+        all_color_modes = list(_find_state_attributes(on_states, ATTR_COLOR_MODE))
+        if all_color_modes:
+            # Report the most common color mode.
+            color_mode_count = Counter(itertools.chain(all_color_modes))
+            self._color_mode = color_mode_count.most_common(1)[0][0]
```

Comment:
This could maybe be a little bit smarter: If group member's `color_mode` is either of `COLOR_MODE_COLOR_TEMP`, `COLOR_MODE_HS`, `COLOR_MODE_RGB`, `COLOR_MODE_RGBW`, `COLOR_MODE_RGBWW`, select the `color_mode` using the most channels